tests/opentracing_instrumentation/test_postgres.py

```diff
@@ -0,0 +1,112 @@
+# Copyright (c) 2018 Uber Technologies, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+from builtins import object
+import random
+
+from basictracer import BasicTracer
+from basictracer.recorder import InMemoryRecorder
+import opentracing
+from opentracing.ext import tags
+
+from opentracing_instrumentation.client_hooks import _dbapi2
+from opentracing_instrumentation.client_hooks import psycopg2
+
+from sqlalchemy import (
+    Column,
+    ForeignKey,
+    Integer,
+    MetaData,
+    String,
+    Table,
+    create_engine,
+)
+from sqlalchemy.orm import mapper, sessionmaker
+
+import pytest
+
+
+@pytest.fixture
+def engine():
+    try:
+        yield create_engine('postgresql://localhost/travis_ci_test')
+    except:
+        pass
+
+
+@pytest.fixture
+def session():
+    Session = sessionmaker()
+    Session.configure(bind=engine)
+    try:
+        yield Session()
+    except:
+        pass
+
+
+@pytest.fixture(autouse=True)
+def patch_postgres():
+    psycopg2.install_patches()
+
+
+@pytest.fixture()
+def tracer():
+    t = BasicTracer(recorder=InMemoryRecorder())
+    old_tracer = opentracing.tracer
+    opentracing.tracer = t
+
+    try:
+        yield t
+    except:
+        opentracing.tracer = old_tracer
+
+
+def check_span():
+    print([span.__dict__ for span in opentracing.tracer.recorder.get_spans()])
+    print(span.tags)
+    assert span.tags[tags.SPAN_KIND] == tags.SPAN_KIND_RPC_CLIENT
+
+
+metadata = MetaData()
+
+user = Table('user', metadata,
+             Column('id', Integer, primary_key=True),
+             Column('name', String(50)),
+             Column('fullname', String(50)),
+             Column('password', String(12)))
+
+
+class User(object):
+
+    def __init__(self, name, fullname, password):
+        self.name = name
+        self.fullname = fullname
+        self.password = password
+
+
+mapper(User, user)
```

Comment:
please move up next to the other vars

Reply:
OK

Reply:
Actually I can't move it because the class is only defined immediately before this. I blame the SQLAlchemy API for the mess of variables in module scope.